src/backend/aspen/api/views/users.py

```diff
@@ -30,3 +34,27 @@ async def update_user_info(
         )
     await db.commit()
     return UserMeResponse.from_orm(user)
+
+
+# Requires prior auth0 account for the new user.
+@router.post("/new")
```

Comment:
I think this is just `@router.post("/")` since the "new" is implied by the `POST`

Reply:
fixed!